.github/composite-actions/download-libraries/action.yml

```diff
@@ -22,7 +22,10 @@ runs:
       shell: bash
 
     - name: Install Boost
-      run: sudo apt-get install libboost-all-dev -y
+      run: |
+        sudo add-apt-repository ppa:mhier/libboost-latest
+        sudo apt update
+        sudo apt-get install libboost1.81-all-dev -y
```

Comment:
It'll probably work fine, but installing latest version from sources is a better approach IMO, because this PPA is unofficial by some completely unknown third-party and we can't actually trust it

Reply:
I'm having trouble building boost from source, the wheel workflows seem to be failing for some reason that I can't understand right now.

Reply:
Figured it out: the build files were cached along with the library and the build script erroneously detected the presence of compression libraries on the wheel container when they weren't there.